hub/src/test/java/ch/prevo/open/hub/nodes/NodeServiceTest.java

```diff
@@ -149,7 +150,7 @@ public void currentEmploymentCommencements() {
     public void notifyMatch() {
         when(nodeRegistry.getCurrentNodes()).thenReturn(asList(node1_new, node2_old));
 
-        EncryptedData transferInformation = new EncryptedData("", "");
+        EncryptedData transferInformation = new EncryptedData(RandomUtils.nextBytes(1), RandomUtils.nextBytes(10));
```

Comment:
Tests should be deterministic, therefore one should try to avoid random input data.